pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchgit, cmake, boost169, zlib }:
+
+let
+  version = "8.9.2";
+in
+stdenv.mkDerivation {
+  pname = "vowpal-wabbit";
+  version = "${version}";
+  src =
+    fetchgit {
+      url = "https://github.com/VowpalWabbit/vowpal_wabbit.git";
+      rev = "${version}";
+      sha256 = "0np11nvxvbxgfdlm4q07lw7zni6mm4l93bf7pj0rif7fz15k8f3l";
+      fetchSubmodules = true;
+    };
+  buildInputs = [
+    cmake
+    boost169
+    zlib
+  ];
+  configurePhase = ''
+    cmake . -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=$out \
+      -DVW_INSTALL=ON \
+      -DBUILD_TESTS=OFF \
+      -DBUILD_JAVA=OFF \
+      -DBUILD_PYTHON=OFF \
+      -DUSE_LATEST_STD=ON
+  '';
+  buildPhase = ''
+    cmake --build . --target all --parallel $NIX_BUILD_CORES
+  '';
+  installPhase = ''
+    cmake --install .
+  '';
+  meta = with lib; {
+    description = "Machine learning system which pushes the frontier of machine learning with techniques such as online, hashing, allreduce, reductions, learning2search, active, and interactive learning";
```

Comment:
This looks much more suited for `meta.longDescription`, AFAIK `meta.description` should be 90 characters or less.